### PR TITLE
Sine and GUI

### PR DESCRIPTION
**Pull request by yongsuH**
It's implemented by taylor series. I've implemented factorial, but power is done by inner function. It should be replaced by our own functions later. 
GUI works as simple calculator, inner functions used should be changed later.

### CODE REVIEW

**Comment by steveiaco** [comment]
The GUI is cool, i'll try testing it out later today or tomorrow. But I thought we were sticking to a command line calculator?

**Comment by yongsuH** [comment]
ah, we can use any of them, this gui calculater is actually finished

File: sin.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    # program will crush if num too big
+    # so minus 2pi continuously, since sin(x+2pi)=sin(x)
+    # this step is losing precision and very slow when num is big, like x=10^9
+    while num > twopi:
+        num -= twopi
+    while num < -twopi:
+        num += twopi
+    # using for loop to create summation i=0 to i=39
+    for i in range(40):
+        res += ((-1)**i)*(num**(2*i+1))/(factorial(2*i+1))
```
Comment (steveiaco) [inline sin.py:28]: We can't use ** as they're not primative arithmetic functions, you can import the power_function and use that

**Review by seanheinrichs — COMMENTED** [review]
I thought we decided that we were not going to use GUI but rather keep this as a CLI for the first iteration? Also is this a copy paste from somewhere? I don't know if it's a good idea to merge a bunch of code we didn't write... thoughts?

**Pull request closed without merging**